src/pages/404.js

```diff
@@ -1,21 +1,27 @@
 import React from "react"
 import styled from "styled-components"
 
+import Translation from "../components/Translation"
 import Link from "../components/Link"
 import { Page, Content } from "../components/SharedStyledComponents"
 
 const StyledPage = styled(Page)`
   margin-top: 4rem;
 `
 
-// TODO is there a way to translate this page?
 const NotFoundPage = () => (
   <StyledPage>
     <Content>
-      <h1>Resource not found.</h1>
+      <h1>
+        <Translation id="page-404-not-found" />
+      </h1>
       <p>
-        Try using search to find what you're looking for or{" "}
-        <Link to="/en/">return home</Link>.
+        <Translation id="page-404-copy" />
+      </p>
+      <p>
+        <Link to="/">
+          <Translation id="home" />
```

Comment:
If we're going to use these translation ID's, we need to add corresponding strings to the translation JSON file. In this case, I would recommend adding them to the `src/intl/en/common.json` file. 
ie:
```
"page-404-not-found": "Resource not found.",
"page-404-copy": "Try using search to find what you're looking for.",
"home": "Return home",
```

On this note, @samajammin were we intentionally not translating this page for any reason?

Reply:
It looks like there are already translations for "page-404-not-found" and "page-404-copy" in 25 of the files though there are only 4 files with translations for "home" and no 'return home' as a separate translation. Wondering if this provides value or if its better to wait for complete translations.

Reply:
Hm, good call. I spot checked it using Romanian, and that one happened to not have those IDs.

@samajammin Looking close, it appears it's the version 2.0, 2.2 and 1.0 pages that don't have these strings anymore. English (v2.4) has them, and v1.1 have them still. The `en/common.json` file for example has 202 items, yet the `ro/common.json` and `es/common.json` files only have 192 items (of the ones missing are these 404 strings).
Any idea why that is?

Reply:
So it's been a bit since I looked into this but I'm not sure translating the 404 page is actually possible. Since the requested page doesn't exist, the application isn't aware what the language is. I don't believe there's an `intl` object in the page's context. Perhaps we could add some logic into the component to inspect the `pathname` & lookup the appropriate string based on that? Might be best to have this as a standalone issue/PR.